### PR TITLE
Added rules for online journals and newspapers

### PR DESCRIPTION
**Pull request by abeeken**

### CODE REVIEW

File: university-of-lincoln-harvard.csl [hunk 1]
```diff
@@ -24,7 +24,7 @@
   </locale>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name and="text"/>
+      <name/>
```
Comment (adam3smith) [inline university-of-lincoln-harvard.csl]: Why this change? I think the and should stay and you should add `initialize-with="."`, no? Or what are you trying to do?

File: university-of-lincoln-harvard.csl [hunk 2]
```diff
@@ -223,31 +220,27 @@
           </group>
         </else-if>
         <else-if type="book graphic report" match="any">
-          <group delimiter=". " prefix=" ">
+          <group prefix=" " delimiter=". ">
             <text variable="note"/>
```
Comment (adam3smith) [inline university-of-lincoln-harvard.csl]: I know this isn't yours, but we want the note removed here. It makes no sense.

File: university-of-lincoln-harvard.csl [hunk 3]
```diff
@@ -223,31 +220,27 @@
...
-            <text macro="title" prefix=" "/>
-            <text macro="edition" prefix=", "/>
-          </group>
+          <text macro="title" prefix=" " suffix="."/>
           <group prefix=" " delimiter=" ">
             <text term="in" text-case="capitalize-first" suffix=":"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix="."/>
             <text variable="collection-title" suffix="."/>
+            <text macro="edition" prefix=" "/>
```
Comment (adam3smith) [inline university-of-lincoln-harvard.csl]: No need for the prefix given the group delimiter, though you may need a suffix to separate edition from date

File: university-of-lincoln-harvard.csl (hunk 1 above)
Comment (abeeken) [reply university-of-lincoln-harvard.csl]: Hi Adam,

I've kind of inherited this work so I'm picking up a lot of things here as I'm going. Pretty much all of these aren't changes I'm sure I've made so I'm not entirely sure why they're needed or what implications they have! This being a prime example, I'm not sure what initialize-with would do for us.

File: university-of-lincoln-harvard.csl (hunk 2 above)
Comment (abeeken) [reply university-of-lincoln-harvard.csl]: Will remove this.

File: university-of-lincoln-harvard.csl (hunk 3 above)
Comment (abeeken) [reply university-of-lincoln-harvard.csl]: I see - I'll get rid of that and take a look at if we need the suffix.

File: university-of-lincoln-harvard.csl (hunk 1 above)
Comment (adam3smith) [reply university-of-lincoln-harvard.csl]: `<name/>` gives you 
Adam Smith, Karl Marx, Andrew Beeken.

`<name and="text" initialize-with="."/>`
gives you A. Smith, K. Marx, and A. Beeken which I think is correct for editors in the style, no?

**Comment by abeeken** [comment]
@adam3smith I've made the recommended changes - hopefully that's all okay!
